articles/tokens/refresh-token/current/index.md

```diff
@@ -234,23 +280,21 @@ If you try to do a <a href="/rules/redirect">redirect</a> with <code>context.red
 
 ### Web Apps
 
-All our main SDKs support Refresh Tokens out of the box. Some are [Node.js](/quickstart/webapp/nodejs), [ASP.NET Core](/quickstart/webapp/aspnet-core), [PHP](/quickstart/webapp/php), [Java](/dev-centers/java), and many more. For a complete listing refer to our [Quickstarts page](/quickstart/webapp).
+All our main SDKs support Refresh Tokens out of the box. Some are [Node.js](/quickstart/webapp/nodejs), [ASP.NET Core](/quickstart/webapp/aspnet-core), [PHP](/quickstart/webapp/php), [Java](/dev-centers/java), and many more. For a complete listing, see [Quickstarts](/quickstart/webapp).
 
 ### Single Page Apps
 
-For web apps that execute on the browser, the way to refresh a token is using [Silent Authentication](/api-auth/tutorials/silent-authentication). [Auth0.js](/libraries/auth0js), our client-side library, provides methods for this out of the box.
+[Silent Authentication](/api-auth/tutorials/silent-authentication) is the method that is used to refresh a token for web apps that execute in a browser. [Auth0.js](/libraries/auth0js), our client-side library, provides methods for this functionality:
 
-- The `authorize` method, redirects the user to the `/authorize` endpoint, in order to login and provide consent.
+- The `authorize` method, redirects the user to the `/authorize` endpoint, to log in and provide consent.
 - The `parseHash` method, parses a URL hash fragment to extract the result of an Auth0 authentication response.
 - The `checkSession` method, attempts to get a new token from Auth0, using [silent authentication](/api-auth/tutorials/silent-authentication). For more details refer to [Using checkSession to Acquire New Tokens](/libraries/auth0js#using-checksession-to-acquire-new-tokens).
 
-More information on the library:
-- [Auth0.js Reference](/libraries/auth0js)
-- [Auth0.js GitHub repo](https://github.com/auth0/auth0.js#api)
+You can find additional information on the library in its [GitHub repository](https://github.com/auth0/auth0.js#api).
```

Comment:
I would assume we'd prefer to steer people to the docs rather than the GitHub, for more info - especially because I think the new SDKs lead is in favor of lowering the amount of README info and directing people to the docs.